birdhouse/config/jupyterhub/config/proxy/conf.extra-service.d/jupyterhub.conf.template

```diff
@@ -1,5 +1,5 @@
     location /jupyter/ {
-        proxy_pass http://${PAVICS_FQDN}:8800/jupyter/;
+        proxy_pass http://jupyterhub:8000/jupyter/;
```

Comment:
Bypasses the Magpie/Twitcher auth. 
Is JupyterHub sufficient by itself to form the Magpie login through the UI?

Reply:
Yes jupyter can handle its own authentication by communicating with magpie directly. I'd love to figure out how to put jupyterhub behind twitcher eventually but I spent a few days trying to figure it out and didn't come up with a good solution so I put it back on the to-do list for later. 

(also, I didn't make any changes to which services are behind twitcher in this PR, see my comment above)

Reply:
To Do: https://github.com/bird-house/birdhouse-deploy/issues/334

Reply:
> Bypasses the Magpie/Twitcher auth.

@fmigneault FYI Jupyter has never been behind Twitcher since day one, it has its own authentication.